joomla/jissues
# Post test results with comment

This is the PR will allow to post test results with comment. Ref #561. 
## Testing instructions
1. Got to [test issue](http://jtracker.j4devs.ru/tracker/jtests/15) on my test website.
2. Try to post different test results with comment.
   2.1 Check that "Tests" section (in the top right column) is also updated according to your test result.
   2.2 Check that your test activity is displayed.
   2.3 Refresh the page and ensure that the test values are correct and your test activity is still there.
3. Try to post different test results using "Submit test result" button.
   3.1 Check that "Tests" section and test radio buttons (under the comment textarea) are also updated according to your test result.
   3.2 Check that your test activity is displayed.
   3.3 Refresh the page and ensure that the test values are correct and your test activity is still there.
